source/hdev/Applications/Advanced/ObjectMatching/SurfaceMatchingFindModel.hdev

```diff
@@ -52,7 +52,7 @@
 <c></c>
 <c>* Configuring HDR settings</c>
 <c>* Twinings tea box</c>
-<l>if (Filename = 'Twinings tea box')</l>
+<l>if (Filename = 'TwiningsTeaBox')</l>
 <l>    set_framegrabber_param (AcqHandle, 'MergedMultiFrameMode','HDR')</l>
```

Comment:
add space between',' and 'HDR', e.g. 'set_framegrabber_param (AcqHandle, 'MergedMultiFrameMode', 'HDR')'.
Need to do this with all others set_framegrabber_param (...) function calls as well.

Reply:
Now I understand what made people make linters. Done.